src/overlays/actors/ovl_En_Viewer/z_en_viewer.c

```diff
@@ -71,51 +112,839 @@ const ActorInit En_Viewer_InitVars = {
     (ActorFunc)EnViewer_Update,
     (ActorFunc)EnViewer_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Viewer/func_80B2A200.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Viewer/EnViewer_Init.s")
+static InitChainEntry sInitChain[] = {
+    ICHAIN_F32(uncullZoneScale, 300, ICHAIN_STOP),
+};
+
+struct_80B2CEE8 D_80B2CEE8[] = {
+    { OBJECT_HORSE_ZELDA, OBJECT_HORSE_ZELDA, 1, 0, 2, 20, 1, &D_06006B2C, &D_06007148 },
+    { OBJECT_IM, OBJECT_OPENING_DEMO1, 1, 0, 0, 10, 3, &D_0600F788, &D_060029CC },
+    { OBJECT_ZL4, OBJECT_OPENING_DEMO1, 1, 0, 0, 10, 2, &D_0600E038, &D_06000450 }, 
+    { OBJECT_GNDD, OBJECT_GNDD, 1, -6, 0, 10, 0, &D_060119E8, &D_06002928 },
+    { OBJECT_HORSE_GANON, OBJECT_HORSE_GANON, 1, 0, 2, 20, 1, &D_06008668, &D_06003858 },
+    { OBJECT_GNDD, OBJECT_GNDD, 1, -6, 0, 10, 0, &D_060119E8, &D_060005B4 },
+    { OBJECT_HORSE_GANON, OBJECT_HORSE_GANON, 1, 0, 2, 20, 1, &D_06008668, &D_06002650 },
+    { OBJECT_GNDD, OBJECT_GNDD, 1, -6, 0, 10, 0, &D_060119E8, &D_06004260 },
+    { OBJECT_GNDD, OBJECT_GNDD, 1, -6, 0, 10, 0, &D_060119E8, &D_060050A8 },
+    { OBJECT_GANON, OBJECT_GANON, 1, -6, 0, 10, 0, &D_060114E8, &D_06011348 },
+};
+
+EnViewerAnimFunc sAnimFuncs[] = {
+    func_80B2A300,
+    func_80B2A4D8,
+    func_80B2A300,
+    func_80B2A448,
+};
+
+static void* ActorShadowDrawFunc[] = {
+    NULL,
+    ActorShadow_DrawFunc_Circle,
+    ActorShadow_DrawFunc_Squiggly,
+};
+
+// timer
+s16 D_80B2CFCC = 0;
+
+Vec3f D_80B2CFD0 = { 0.0f, 0.0f, 0.0f };
+
+static EnViewerDrawFunc sDrawFuncs[] = {
+    func_80B2B4A8,
+    func_80B2B8FC,
+    func_80B2BA38,
+    func_80B2C130,
+};
+
+// angle
+s16 D_80B2CFEC = 0;
+
+EnGanonMant* ganonCape;
```

Comment:
s prefix

Reply:
Even though it's not static?

Reply:
s and g are not directly related to something being `static` or not, its more of a way to label the variables scope in the project. `s` is for local, `g` is for global

Reply:
and this var will become static after all of the functions match anyway